synapse/cortex.py

```diff
@@ -1231,6 +1177,8 @@ def pokeFormCount(self, form, valu):
     async def _calcFormCounts(self):
         '''
         Recalculate form counts from scratch.
+
+        Note:  this only counts nodes in the main view
```

Comment:
recalculating this is will only report values from the main view's top layer; but actual run time values will account for additions and subtractions from all views.

Reply:
Fixed it so that run time values only applied when on main view's write layer.